chain/epoch-manager/src/proposals.rs

```diff
@@ -45,10 +45,10 @@ pub fn proposals_to_epoch_info(
     validator_kickout: HashMap<AccountId, ValidatorKickoutReason>,
     validator_reward: HashMap<AccountId, Balance>,
     minted_amount: Balance,
+    current_version: ProtocolVersion,
     next_version: ProtocolVersion,
```

Comment:
Nothing like changing the order to two arguments with the same type 😈 😄 

On a serious note, this is a good change but the compiler won't tell you if you forget to update it somewhere. Please make sure you grep for all usages of this function and make sure to correctly set the args.

Reply:
Yeah, I've checked all 17 usages. Funny enough, in most of them the same protocol version is given